hack/observability/kube-state-metrics/chart/values.yaml

```diff
@@ -0,0 +1,77 @@
+image:
+  tag: v2.6.0
```

Comment:
Would be good if we can drop the pinned tag once the corresponding Helm chart is out
(just that when we pull the latest chart the pinned tag and the chart don't run out of sync)

Reply:
I agree, the latest chart still refers 2.5.0

Reply:
Let's open an issue to track this

Reply:
done: #7143